components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/listener/SCIMUserOperationListener.java

```diff
@@ -221,6 +227,8 @@ public boolean doPreSetUserClaimValuesWithID(String userID, Map<String, String>
             throw new UserStoreException("Error while reading isScimEnabled from userstore manager", e);
         }
 
+        validateProvisionedUserClaimUpdate(userID, profileName, userStoreManager);
```

Comment:
Need to samething for doPreSetUserClaimValues, doPreSetUserClaimValue, doPreSetUserClaimValueWithID

Reply:
Fixed with https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/pull/367/commits/f4e735453ad2ac354f66a24d63100cecac594701